### ISSUE TITLE
output

### ISSUE THREAD

**Issue by nmskhan** [GT-EmoryMINDlab/rodent-whole-brain-preprocessing-recipe#6]
Hello Sam
Thanks for sharing your codes and it was very useful for us to try your sample data before testing our data.  One question, I have is, what is the output file of the final processing. Is there any way to display the resting-state connectivity on the images in addition to the FC map? Please let me know. 
Thanks
Ned

**Comment by imnanxu** [comment]
Hi Ned,

I apologize for the delay in response. As is described in the README
(section 4.3.5), the two final outputs are _mc_topup_norm_fil_reg_sm (for
the spatially smoothed brain) and _mc_topup_norm_fil_reg_sm_seed.txt (for
the seed extracted timeseries). The FC map was obtained by computing the
correlation of the seed extracted timeseries. You cannot display pairwise
connectivity on brain images as they have different dimensions. The
pairwise connectivity depicts the connections between different regions.

I hope it helps!
Nan

**Comment by nmskhan** [comment]
Hi Nan
Thanks for your reply. Thank you for putting together a very long scripts for fmri preprocessing. One quick question, how to compute FC map from the seed extracted series? I tried to run your sample data, I was able to complete it without error but did not get FC map. Do you suggest any software tool to convert the seed txt into FC correlation map?
By the way, I am from Georgia State university.
Thank you very much
Ned

**Comment by imnanxu** [comment]
Hi Ned,

Yes, the pipeline is only for preprocessing whereas the FC map computation
is usually considered as a step of post-analysis. You can obtain the FC map
by running the following two commands in Matlab:

Mat=dlmread('EPI_mc_topup_norm_fil_reg_sm_seed.txt');
FC=corr(Mat);

Hope it helps!
Nan

**Comment by nmskhan** [comment]
Thank you very much! Nan
This will help me a lot.
Thanks
Ned

**Comment by nmskhan** [comment]
Hi Nan
One quick question, what was the actual orientation you sample (mice) data collected? Is it transaxial or coronal (horizontal)? 
When I collected the mouse data in axial, I had to swap dimension/orientation (using fslswapdim), to bring similar to your atlas file. The script1 works but script 2 stopped, when it was calling AFNI script, (afni script do not recongnizes the fsl file, because it has no s and q form (no sptial transformation) due to dimension swap i believe). That is why I am wondering what orientation do i have to collect without making any changes in orienation in later step.

Another work around would be changing orienation of atlas and mask files (lib/tmp) without touching  mouse data files.

Any suggestions.
Thanks
Khan
